Patches/Moyo from the depth/Ammo/Ammo_Moyo.xml

```diff
@@ -320,6 +320,103 @@
 				</value>
 			</li>
 			
+			<!-- ==================== Abysteel Nail ========================== -->
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs</xpath>
+				<value>
+					<!-- ==================== AmmoSets ========================== -->
+
+					<CombatExtended.AmmoSetDef>
+						<defName>AmmoSet_Abyssnail</defName>
+						<label>Moyo alloy nail</label>
+						<ammoTypes>
+							<Ammo_Abyssnail>Bullet_Abyssnail</Ammo_Abyssnail>
+						</ammoTypes>
+					</CombatExtended.AmmoSetDef>
+					
+					<CombatExtended.AmmoCategoryDef>
+						<defName>MoyoNail</defName>
+						<label>Nail</label>
+						<description>High velocity alloy nail.</description>
+					</CombatExtended.AmmoCategoryDef>
+
+					<!-- ==================== Ammo ========================== -->
+
+					<ThingDef Class="CombatExtended.AmmoDef" ParentName="SpacerSmallAmmoBase">
+						<defName>Ammo_Abyssnail</defName>
+						<label>Abyssteel Nail</label>
+						<description>Machined nails made of abyssteel.</description>
+						<statBases>
+							<Mass>1.02</Mass>
+							<Bulk>2.87</Bulk>
+							<MarketValue>27.22</MarketValue>
+						</statBases>
+						<tradeTags>
+							<li>CE_AutoEnableTrade</li>
+							<li>CE_AutoEnableCrafting_TableMachining</li>
+						</tradeTags>
+						<thingCategories>
+							<li>AmmoAdvanced</li>
+						</thingCategories>
+						<stackLimit>500</stackLimit>
+						<graphicData>
+							<texPath>ThirdParty/Rimsenal/Nail</texPath>

```

Comment:
I wouldn't mind a different, more spacer-looking ammo sprite. The nail is... pretty crude.